source/examples/verovio/SchulhoffSonata.mei

```diff
@@ -0,0 +1,100 @@
+          </useRestrict>
+        </availability>
+      </pubStmt>
+      <seriesStmt>
+        <title>MEI Guidelines examples</title>
+      </seriesStmt>
+    </fileDesc>
+  </meiHead>
+  <music>
+    <body>
+      <mdiv n="1" label="Schulhoff's Violin Sonata" xml:id="Schulhoff">
+        <score>
+          <scoreDef system.leftline="false">
+            <staffGrp>
+              <staffDef n="1" clef.line="2" clef.shape="G" lines="5" />
+            </staffGrp>
+          </scoreDef>
+          <section>
+            <measure right="single">
+              <staff n="1">
+                <layer>
+                  <beam>
+                    <note dur="8" oct="4" pname="a" xml:id="note1">
+                      <accid accid="n" />
+                      <artic artic="stacc acc" />
+                    </note>
+                    <note dur="8" oct="4" pname="b" xml:id="note2">
+                      <artic artic="stacc acc" />
+                    </note>
+                    <note dur="8" oct="5" pname="c" xml:id="note3">
+                      <accid accid="n" />
+                      <artic artic="stacc acc" />
+                    </note>
+                    <note dur="8" oct="4" pname="d" xml:id="note4">
+                      <accid accid="s" />
+                      <artic artic="stacc acc" />
+                    </note>
+                  </beam>
+                  <note dur="8" oct="4" pname="e" xml:id="note5">
+                    <artic artic="stacc acc" />
+                  </note>
+                  <note dur="8" oct="4" pname="f" xml:id="note6">
+                    <accid accid="s" />
+                    <artic artic="stacc acc" />
+                  </note>
+                </layer>
+              </staff>
+              <?edit-start?>
+              <beamSpan startid="#note5" endid="#note8" plist="#note5 #note6 #note7 #note8" />
+              <?edit-end?>
+            </measure>
+            <measure right="single">
+              <staff n="1">
+                <layer>
+                  <note dur="8" oct="4" pname="g" xml:id="note7">
+                    <accid accid="n" />
+                    <artic artic="stacc acc" />
+                  </note>
+                  <note dur="8" oct="3" pname="a" xml:id="note8">
+                    <accid accid="n" />
+                    <artic artic="stacc acc" />
+                  </note>
+                  <beam>
+                    <note dur="8" oct="3" pname="b" xml:id="note9">
+                      <accid accid="f" />
+                      <artic artic="stacc acc" />
+                    </note>
+                    <note dur="8" oct="4" pname="c" xml:id="noteA">
+                      <accid accid="n" />
+                      <artic artic="stacc acc" />
+                    </note>
+                  </beam>
+                  <note dur="4" oct="4" pname="d" xml:id="noteB">
+                    <accid accid="f" />

```

Comment:
Hexadecimal numbering of the note ids could confuse people with the actual pname :D